modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= S3 - Compatible backup storage providers
+
+
+OADP supports a variety of object storage providers for different backup and snapshot operations.
+
+== S3 drivers for Backup Storage Locations
+
+=== Supported
+
+The following S3 compatible object storage providers, are fully supported by OADP through AWS plugin, for use as Backup Storage Locations:
+
+* Minio
+* MCG / nooba
+* AWS S3
+
+[NOTE]
+====
+The following compatible object storage providers are supported and have their own Velero object store plugins:
+
+* Google Cloud Platform (GCP)
+* Microsoft Azure
+====
+
+=== Unsupported
+
+The following S3 compatible object storage providers, are known to work with Velero through AWS plugin, for use as Backup Storage Locations, but are unsupported and have not been tested by Red Hat:
+
+* IBM Cloud
+* Oracle Cloud
+* Minio
+* DigitalOcean
+* NooBaa
+* Tencent Cloud
+* Ceph RADOS v12.2.7
+* Quobyte
+* Cloudian HyperStore
+
+=== Known issues
+
+The following S3 compatible object storage providers, are known not work with OADP through AWS plugin, for use as Backup Storage Locations:
```

Comment:
The text is not quite correct here. I would say this instead:
"The following S3 compatible object storage providers, are known to work with Velero through AWS plugin with a limited feature set. They work for use as Backup Storage Locations for backup storage, but are not compatible with Restic for filesystem-based volume backup and restore."

Reply:
@sseago @weshayutin @kaovilai @tareqalayan Do you have any other feedback?

Reply:
@CarmiWisemon Looks good to me now.